core/DCache.hpp

```diff
@@ -81,6 +121,38 @@ namespace olympia
         ////////////////////////////////////////////////////////////////////////////////
         // Events
         ////////////////////////////////////////////////////////////////////////////////
+        sparta::UniqueEvent<> uev_mshr_request_{
+            &unit_event_set_, "mshr_request", CREATE_SPARTA_HANDLER(DCache, mshrRequest_)};
+
+
+        sparta::utils::ValidValue<MemoryAccessInfoPtr> l2_mem_access_info_;
+        sparta::utils::ValidValue<MemoryAccessInfoPtr> lsu_mem_access_info_;
+        void arbitrate_l2_lsu_req_()
```

Comment:
Can you change the function name to match the coding style?

Reply:
My bad I should have fixed it before, I will create a PR to add this linting style to the github actions.